index.js

```diff
@@ -1,14 +1,21 @@
 const compiler = require('vue-template-compiler');
 const transpile = require('vue-template-es2015-compiler');
+const parse5 = require('parse5');
+const consolidate = require('consolidate');
+const spawnSync = require('child_process').spawnSync;
+const pathJoin = require('path').join;
 
 const exportsTarget = '((module.exports.default || module.exports).options || module.exports.default || module.exports)';
 
 module.exports = function ({ content, filename, sourceMap }) {
-  const startOfScript = content.indexOf('<script>') + 8;
-  const endOfScript = content.lastIndexOf('</script>');
-  if (startOfScript < 8 || endOfScript < 0 || endOfScript < startOfScript){
+  const documentFragment = parse5.parseFragment(content, {locationInfo: true});
```

Comment:
I stand corrected. I swapped parse5 out for some regular expressions to pull out script/template content and extract their attributes, and the above tests went up to **35.776**!